packages/yourloops/pages/signup/signup-consent.tsx

```diff
@@ -151,12 +155,13 @@ export default function SignUpConsent(props: SignUpFormProps): JSX.Element {
           }
           label={t(
             `signup-consent-${state.formValues.accountRole}-terms-condition`,
-            { terms: "Terms or Use" }
+            { terms: t("Terms of Use") }

```

Comment:
```suggestion
            { terms: t("terms-and-conditions") }
```
They key name is not good, but the translation is.